tests/countries/test_cape_verde.py

```diff
@@ -0,0 +1,109 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.constants import OPTIONAL, PUBLIC
+from holidays.countries import CapeVerde, CV, CAV
+from tests.common import CommonCountryTests
+
+
+class TestCapeVerde(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(CapeVerde, years=range(1973, 2050))
+
+    def test_country_aliases(self):
+        self.assertAliases(CapeVerde, CV, CAV)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(CapeVerde(years=1972))
+
+    def test_2025_holidays(self):
+        self.assertHolidays(
+            CapeVerde(years=2025),
+            ("2025-01-01", "Ano Novo"),
+            ("2025-01-13", "Dia da Liberdade e Democracia"),
+            ("2025-01-20", "Dia da Nacionalidade e dos Heróis Nacionais"),
+            ("2025-04-18", "Sexta-feira Santa"),
+            ("2025-05-01", "Dia do Trabalhador"),
+            ("2025-06-01", "Dia Mundial da Criança"),
+            ("2025-07-05", "Dia da Independência Nacional"),
+            ("2025-08-15", "Dia da Assunção"),
+            ("2025-11-01", "Dia de Todos os Santos"),
+            ("2025-12-25", "Natal"),
+        )
```

Comment:
Put `test_2025_public_holidays` at the end of the file instead, before `test_default_l10n` and `test_en_us_l10n`.

For `test_[insert]_l10n` ones, see [test_macau.py](https://github.com/vacanza/holidays/blob/dev/tests/countries/test_macau.py#L1102) for example